docs/monitoring/autodiscovery.md

```diff
@@ -90,6 +90,30 @@ In container environments you add the `MONITORING_TOKEN` environment variable, o
 
 In all environments this makes it possible to use different monitoring tokens, and thus different Sematext Apps for the same service type running on the same machine or in the same Kubernetes/Swarm cluster.
 
+### Switching from manual to automatic monitoring
+
+The switch from manual to automatic monitoring is done by first removing manually set up agent and then enabling automatic monitoring in the user interface.
```

Comment:
The switch from manual to automatic monitoring is done by first removing manually set up agent and then enabling automatic monitoring in the user interface.
=>
The switch from manual to automatic monitoring is done by first removing the agent you set up manually, then enabling automatic monitoring in the user interface.